metricflow-semantics/metricflow_semantics/naming/dunder_scheme.py

```diff
@@ -77,8 +77,9 @@ def spec_pattern(self, input_str: str, semantic_manifest_lookup: SemanticManifes
             )
 
         # At this point, len(input_str_parts) >= 2
-        custom_granularity_names: List[str] = []  # TODO
-        valid_granularity_names = [granularity.value for granularity in TimeGranularity] + custom_granularity_names
+        valid_granularity_names = [granularity.value for granularity in TimeGranularity] + list(
+            semantic_manifest_lookup._custom_granularities.keys()
```

Comment:
nit: prefer if we could make `_custom_granularities` method not protected or add another accessor that is not a protected method